### PR TITLE
DO- 437 Business Edit Changes(14 July)

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

File: src/components/member/BusinessDetails/BusinessDetails.jsx [hunk 1]
```diff
@@ -276,13 +291,18 @@ const BusinessPrimaryDetails = (props) => {
     description = 'NA',
     name = 'NA',
     registeredAddress = {},
+    phone,
+    website,
   } = props;
+  console.log(website,phone);
```
Comment (ayushmalik2110) [inline src/components/member/BusinessDetails/BusinessDetails.jsx:297]: Console.log ????

File: src/components/member/BusinessDetails/BusinessLabel/BusinessLabel.jsx [hunk 2]
```diff
@@ -13,11 +14,12 @@ import Label from '../../../common/Typography/Label';
  */
 const BusinessLabel = (props) => {
   const { labelKey, labelValue } = props;
+  console.log(labelValue);
```
Comment (ayushmalik2110) [inline src/components/member/BusinessDetails/BusinessLabel/BusinessLabel.jsx:17]: console.log ???

File: src/components/member/BusinessDetails/BusinessDetails.jsx (hunk 1 above)
Comment (Palakagrawal28) [reply src/components/member/BusinessDetails/BusinessDetails.jsx:297]: Done

File: src/components/member/BusinessDetails/BusinessLabel/BusinessLabel.jsx (hunk 2 above)
Comment (Palakagrawal28) [reply src/components/member/BusinessDetails/BusinessLabel/BusinessLabel.jsx:17]: Done

**Pull request merged**